aiofreepybox/api/home.py

```diff
@@ -0,0 +1,149 @@
+# Home structure : adapter > node > endpoint
+class Home:
+
+    def __init__(self, access):
+        self._access = access
+
+    async def get_adapter(self, adapterid):
+        '''
+        Retrieve a registered HomeAdapter
+        '''
+        return await self._access.get('home/adapters/', adapterid)
+
+    async def get_adapters(self):
+        '''
+        Retrieve the list of registered HomeAdapters
+        '''
+        return await self._access.get('home/adapters/')
+
+    async def get_camera(self):
+        '''
+        Get camera info
+        '''
+        return await self._access.get('camera/')
+
+    async def get_endpoint_value(self, nodeid, endpointid):
+        '''
+        Get endpoint value
+        '''
+        return await self._access.get('home/endpoints/{0}/'.format(nodeid), endpointid)
+
+    async def set_endpoint_value(self, nodeid, endpointid, value):
+        '''
+        Set endpoint value
+        '''
+        self._access.put('home/endpoints/{0}/'.format(nodeid), endpointid, value)
+
+    async def del_link(self, linkid):
+        '''
+        Delete link
+        '''
+        return await self._access.delete('home/links/', linkid)
```

Comment:
I don't have access to the API documentation so I can't be sure but I suspect the `link_id` should be appended to the url here as well

Reply:
fixed